app/src/examples/ScreenAnimationExample.tsx

```diff
@@ -0,0 +1,96 @@
+// @ts-nocheck - It will be removed after release of rn-screens, because currently 'react-native-screens/gesture-handler' import doesn't exist.
```

Comment:
```suggestion
// @ts-nocheck - It will be removed after release of react-native-screens, because currently 'react-native-screens/gesture-handler' import doesn't exist.
```